etc/releasing.md

```diff
@@ -131,103 +124,35 @@ Ensure the list of bundled dependencies in `etc/purls.txt` is up-to-date. If not
 If `etc/purls.txt` was updated, update the SBOM Lite document using the following command(s):
 
 ```bash
-# Artifactory and Silk credentials.
+# Artifactory credentials.
 . $HOME/.secrets/artifactory-creds.txt
-. $HOME/.secrets/silk-creds.txt
 
 # Output: "Login succeeded!"
 podman login --password-stdin --username "${ARTIFACTORY_USER:?}" artifactory.corp.mongodb.com <<<"${ARTIFACTORY_PASSWORD:?}"
 
 # Ensure latest version of SilkBomb is being used.
-podman pull artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:1.1
+podman pull artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:2.0
 
 # Output: "... writing sbom to file"
-podman run \
-  --env-file "$HOME/.secrets/silk-creds.txt" \
-  -it --rm -v "$(pwd):/pwd" \
-  artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:1.1 \
-  update -p "/pwd/etc/purls.txt" -i "/pwd/etc/cyclonedx.sbom.json" -o "/pwd/etc/cyclonedx.sbom.json"
+podman run -it --rm -v "$(pwd):/pwd" silkbomb:2.0 \
+  update --refresh --no-update-sbom-version -p "/pwd/etc/purls.txt" -i "/pwd/etc/cyclonedx.sbom.json" -o "/pwd/etc/cyclonedx.sbom.json"
```

Comment:
I expect the full qualified image URL is needed to avoid a `repository does not exist` error:

```bash

silkbomb="artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:2.0"

podman pull "${silkbomb:?}"

podman run -it --rm -v "$(pwd):/pwd" "${silkbomb:?}" \
  update --refresh --no-update-sbom-version -p "/pwd/etc/purls.txt" -i "/pwd/etc/cyclonedx.sbom.json" -o "/pwd/etc/cyclonedx.sbom.json"
```

Reply:
It seems to depend on whether or not Artifactory is included in local registry configurations (e.g. `/etc/containers/registeries.conf`?). Admittedly I do not fully understand how it works, so I am fine with reverting to the "known to work" approach.